test/browser.rollbar.test.js

```diff
@@ -153,6 +179,24 @@ describe('Rollbar()', function() {
 
     done();
   });
+
+  it('should have a tracer if valid tracer is provided', function(done) {
```

Comment:
Should we also have a test when no tracer object is provided?

Reply:
@vselvarajijay that's the same as the invalid tracer object IMO